src/main/java/org/alliancegenome/curation_api/model/entities/ConditionRelation.java

```diff
@@ -41,7 +42,7 @@ public class ConditionRelation extends BaseGeneratedAndUniqueIdEntity {
     @IndexingDependency(reindexOnUpdate = ReindexOnUpdate.SHALLOW)
     @ManyToOne
     @JsonView({View.FieldsOnly.class})
-    @JsonProperty("single_reference")
+    @JsonProperty("singleReference")
```

Comment:
@cmpich we don't need the @jsonproperty if its going to be the same name as the field in java, plus its better not to remap names anyway.